src/test/resources/org/dita/dost/project/product.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../../../../../../main/resources/project.rnc" type="application/relax-ng-compact-syntax"?>
+<project>
+  <include href="common.xml"/>
+  <deliverable name="name">
+    <context name="Site" id="site">
+      <inputs>
+        <input href="site.ditamap"/>
+      </inputs>
+      <profile>
+        <ditaval href="site.ditaval"/>
+      </profile>
+    </context>
+    <output>./site</output>
+    <publication idref="common-sitePub2"/>
```

Comment:
For now, I'm using this syntax. Later on we can decide if we want to use e.g.

```xml
<publication-ref id="common-sitePub2" />
```

Instead and how we want to handle possible overriding of e.g. parameters.

Reply:
Looks good to me. Do you also plan to add idrefs for "context" elements?